dlp/docs/conf.py

```diff
@@ -0,0 +1,311 @@
+# -*- coding: utf-8 -*-
```

Comment:
We should separately discuss these generated docs configs, unless you want me to get into it in this PR.